wp-content/themes/engage/assets/scss/components/_landing.scss

```diff
@@ -9,13 +9,25 @@
   .hero .container {
     display: grid;
 
-    margin: 100px 0;
+    margin: 70px 0 100px 0;
 
     .quiz-creator-intro-copy {
+      margin-right:10px;
       padding: 25px;
+      .research-link {
```

Comment:
OK - last items on this then it'll be all done. Can you make these CSS classes as "flat" as possible? The lower the specificity, the better. This will compile to:
`.hero .container .quiz-creator-intro-copy .research-link`, when it's possible all we need is `.research-link` (or maybe `.quiz-creator-intro__link`.

Let me know if you have any questions on this. The idea is that the lower the specificity, the easier it is to build off of later (if we need to), so it's good practice to make it a habit now.

Reply:
@sghsri can you take a look at this when you get a chance?

Reply:
Hi sorry for the delay didn't see this till today, but I just flattened the classes and pushed it up